### PR TITLE
project restructure and testcases added

### PR DESCRIPTION
**Pull request by chavarera**
#10 
project restructured
and test cases added from select option

### CODE REVIEW

**Comment by aahnik** [comment]
i did not understand why web driver manager is using github API, and why rate limit is exceeding.

while using web driver manager, can we specify path to chromedriver ? we can install chromedriver manually in the gh action, and add it to path.

if it is in path, I hope webdriver manager would not install it.

**Comment by chavarera** [comment]
webdriver manager using GitHub api to download appropriate chrome driver.

can we use any random GH_TOKEN token gh actions or can we put into GitHub secrets

**Comment by aahnik** [comment]
> can we use any random GH_TOKEN token gh actions or can we put into GitHub secrets

any random value will not work. that is just an example in the readme.

The github public API is free to use, but there are rate limits. I realized that the webdriver manager uses github API always. not just when it runs in github actions.

it uses github API, and google API, to find out the version of the latest release of driver, and whether it is compatible or not.

File: s_tool/driver.py [hunk 1]
```diff
@@ -49,6 +46,9 @@ def __enter__(self):
 
     def _load_driver(self):
         """Create Selenium webdriver object"""
+        # Attaching GH_TOKEN value to resolve github public api issue
+        system("export GH_TOKEN='asdasdahsdat'")
```
Comment (aahnik) [inline s_tool/driver.py:50]: remove this line from code.

the webdriver manager will read the GH_TOKEN from environment. we need to set that in environment variable.

Comment (chavarera) [reply s_tool/driver.py:50]: Sure @aahnik

**Pull request merged**